<?xml version="1.0"?>
<launch>
	<!-- Scanner parameters -->
	<param name="laser_scan_period" value="0.025" type="double" />
	<param name="scanner_frame_id" value="laser_scanner" type="str" />
	
	<!-- Define the transform between the base of the robot and the laser scanner -->
	<node name="laser_scanner_frame_mgr" type="static_transform_publisher" pkg="tf" args="0 0 0 0 0 0 1 /base_link /laser_scanner 10" />
	
	<node name="maidbot_scanner_node" type="maidbot_scanner_node" pkg="maidbot_scanner" output="screen"/>
</launch>
